scss/components/_activity-panel.scss

```diff
@@ -1,16 +1,27 @@
 .civicase__activity-panel {
-  margin-top: 37px;
+  &.affix {
+    position: fixed !important;
```

Comment:
why the important?

Reply:
There is a `relative` position applied to the element, probably coming from `Affix` lib. Honestly I dont have time left today to investigate this. Can we leave this out?

Reply:
@reneolivo could you check please

Reply:
I checked, without the important the position gets replaced constantly by a JS script, probably Affix. Without it, the component breaks.